w-dms/app.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"net/http"
+
+	"flag"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// Task is task
+// swagger:model
+type Task struct {
+	ID           interface{} `json:"id" bson:"_id,omitempty"`
+	Alias        string      `json:"alias" bson:"alias"`
+	Description  string      `json:"desc"`
+	Type         string      `json:"type"`
+	Tags         []string    `json:"tags"`
+	Timestamp    int64       `json:"ts"`
+	EstimateTime string      `json:"etime"`
+	RealTime     string      `json:"rtime"`
+	Reminders    []string    `json:"reminders"`
+}
+
+var (
+	errTaskNotFound = errors.New("Task not found")
+	errUnknownURL   = errors.New("Not found")
+	errMissingID    = errors.New("Task id is missing")
+	errInvalidID    = errors.New("Invalid task id")
+)
+
+type (
+	// TaskList is tasklist
+	TaskList []Task
+
+	dbDriver interface {
+		Create(t Task) error
```

Comment:
Yes, but the interface was specified in task. Please see
https://github.com/GolangUA/workshops/blob/master/dms/app.go